docs/source/about/installation.rst

```diff
@@ -104,17 +104,24 @@ Flatpak Package
    :alt: GitHub issues by-label
 
 #. Install `Flatpak <https://flatpak.org/setup/>`_ as required.
-#. Download ``sunshine.flatpak`` and run the following code.
+#. Download ``sunshine_{arch}.flatpak`` and run the following code.
+
+   .. Note:: Be sure to replace ``{arch}`` with the architecture for your operating system.
 
    System level (recommended)
       .. code-block:: bash
 
-         flatpak install --system sunshine.flatpak
+         flatpak install --system ./sunshine_{arch}.flatpak
 
    User level
       .. code-block:: bash
 
-         flatpak install --user sunshine.flatpak
+         flatpak install --user ./sunshine_{arch}.flatpak
+
+   Additional installation (required)
+      .. code-block:: bash
+
+         flatpak run --command=additional-install.sh dev.lizardbyte.sunshine
```

Comment:
Add this code to each sub section (system and user level) instead of making a new sub section.

And if you agree to rename the scripts, change the name here as well.